pkg/controller/template/render.go

```diff
@@ -481,6 +487,8 @@ func onPremPlatformAPIServerInternalIP(cfg RenderConfig) (interface{}, error) {
 			return nil, nil
 		case configv1.KubevirtPlatformType:
 			return cfg.Infra.Status.PlatformStatus.Kubevirt.APIServerInternalIP, nil
+		case configv1.NutanixPlatformType:
+			return cfg.Infra.Status.PlatformStatus.Nutanix.APIServerInternalIP, nil
```

Comment:
seeing: 
`pkg/controller/template/render.go:491:42: cfg.ControllerConfigSpec.Infra.Status.PlatformStatus.Nutanix undefined (type *"github.com/openshift/api/config/v1".PlatformStatus has no field or method Nutanix)`

Reply:
The api dependency merged from the other PR should have addressed this.